### PR TITLE
a couple of fixes for the autolock setting header

### LINKED ISSUE #205: Tables allow headings and things scroll on top of other cells
I noticed a few little bugs that may be worth prioritizing and cleaning up:

- [x] Settings table section headings remain anchored when scrolling backwards but the text scrolls over top the cells below (design needed on expected UI and interactions)
  img_2492

- [x] Auto Lock blurb ("Select when..") remains anchored when scrolling backwards but the text scrolls over top the cells below (design needed on expected UI and interactions)
  img_2493

-  "Search your entries" entry list filter does not remain anchored and can scroll away from the top title bar (is that expected? design needed on expected UI and interaction)
  img_2494

### PR DESCRIPTION
**Pull request by joeyg**
Resolves #205. I believe the SettingList screen has already been taken care of.

### CODE REVIEW

File: lockbox-ios/View/AutoLockSettingsView.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ class AutoLockSettingView: UITableViewController {
     private var dataSource: RxTableViewSectionedReloadDataSource<AutoLockSettingSectionModel>?
 
     init() {
-        super.init(nibName: nil, bundle: nil)
+        super.init(style: UITableViewStyle.grouped)
```
Comment (sashei) [inline lockbox-ios/View/AutoLockSettingsView.swift]: 😍

**Comment by devinreams** [comment]
Thanks @joeyg. I just tested and confirmed these fixed the two issues I spotted (on my device).

I appreciate you spotting and tackling this. 😁

**Comment by joeyg** [comment]
@devinreams thanks for testing! Whats the best way to get builds for the lockbox app? Is there an enterprise build? I have the Focus app on my buddybuild dashboard but not lockbox.

**Comment by devinreams** [comment]
>Whats the best way to get builds for the lockbox app?

Now that you (just) got added to this repo as a "Collaborator" you should see Lockbox in BuddyBuild. Your fancy badge here in GitHub reflects that, too (from being a "Contributor"). Congrats on the promotion. 😉

**Comment by joeyg** [comment]
Thanks! that was really helpful!

**Pull request merged**